public/js/main.js

```diff
@@ -9,8 +9,8 @@ var listString = '';
 function convertToHTML(list){
   var content = allList.map(function(item){
     return  `
-      <li>
-        <div class="view" id= ${item.id}>
+      <li id= ${item.id} class=${item.done?'completed':''}>
+        <div class="view">
```

Comment:
The class name is very general, you should be more specific.
For example: `todo-wrapper`

- [ ] consider